src/irmin-pack/unix/io_intf.ml

```diff
@@ -86,6 +86,12 @@ module type S = sig
   val mkdir : string -> (unit, [> mkdir_error ]) result
   val unlink : string -> (unit, [> `Sys_error of string ]) result
 
+  val unlink_dont_wait : on_error:(exn -> unit) -> string -> unit
```

Comment:
Nit: it's not technically an error.
```suggestion
  val unlink_dont_wait : on_exn:(exn -> unit) -> string -> unit
```

Reply:
:+1: Done with the https://github.com/mirage/irmin/commit/55e95ea067f441d95edbe09c3c86ff9bb07d5107